packages/clay-autocomplete/stories/index.tsx

```diff
@@ -52,10 +54,23 @@ const ClayAutocompleteWithState = () => {
 	const loading = networkStatus < 4;
 	const error = networkStatus === 5;
 
+	useArrowKeysForList(
+		activeIndex,
+		resource ? resource.results.length : 0,
+		setActiveIndex,
+		inputRef.current
+	);
+
+	useEnterKey(
```

Comment:
I'm thinking maybe this is unnecessary, we can have the same effect by adding `onEnter` to `<ClayAutocomplete.Item /> `and React will handle events... what do you think?